src/main/java/net/jingles/enchantments/Commands.java

```diff
@@ -27,124 +27,148 @@
+          return false;
+        }
 
-      if (held == null) {
-        sender.sendMessage(ERROR + "You must be holding an item to enchant.");
-        return false;
-      }
+        held = getHeldItem(sender);
 
-      CustomEnchant enchant = getCustomEnchant(Arrays.copyOfRange(args, 1, args.length));
+        if (held == null || held.getType() == Material.AIR) {
+          sender.sendMessage(ERROR + "You must be holding an item to unenchant.");
+          return false;
+        }
+        enchant = getCustomEnchant(Arrays.copyOfRange(args, 1, args.length));
 
-      if (enchant == null) {
-        sender.sendMessage(ERROR + "An enchantment with that name could not be found.");
-        return false;
-      }
+        if (enchant == null) {
+          sender.sendMessage(ERROR + "An enchantment with that name could not be found.");
+          return false;
+        }
 
-      ItemMeta meta = held.getItemMeta();
+        meta = held.getItemMeta();
 
-      if (meta == null || !meta.hasEnchant(enchant)) {
-        sender.sendMessage(ERROR + "This tem does not have that enchantment!");
-        return false;
-      }
+        if (meta == null || !meta.hasEnchant(enchant)) {
+          sender.sendMessage(ERROR + "This item does not have that enchantment!");
+          return false;
+        }
 
-      meta.removeEnchant(enchant);
+        meta.removeEnchant(enchant);
 
-      if (meta.getLore() != null) {
-        List<String> lore = meta.getLore();
-        lore.removeIf(line -> line.contains(enchant.getName()));
-        meta.setLore(lore);
-      }
+        if (meta.getLore() != null) {
+          lore = meta.getLore();
+          lore.removeIf(line -> line.contains(enchant.getName()));
+          meta.setLore(lore);
+        }
 
-      held.setItemMeta(meta);
-      sender.sendMessage(TITLE + "Successfully removed enchantment.");
+        held.setItemMeta(meta);
+        sender.sendMessage(TITLE + "Successfully removed enchantment.");
+        break;
 
-    } else if (args[0].equalsIgnoreCase("list")) {
+      case "list":
 
-      String list = ChatColor.AQUA + Enchantments.REGISTERED.stream()
-              .map(CustomEnchant::getName).collect(Collectors.joining(", "));
-      sender.sendMessage(TITLE + "Registered custom enchantments: " + list);
+        String list = ChatColor.AQUA + Enchantments.REGISTERED.stream()
+                .map(CustomEnchant::getName).collect(Collectors.joining(", "));
+        sender.sendMessage(TITLE + "Registered custom enchantments: " + list);
+        break;
 
-    }
+      case "info":
+
+        if (args.length < 2) {
+          sender.sendMessage(ERROR + "Incorrect arguments: /enchantments info <enchantment name>");
+          return false;
+        }
+
+        try {
+        enchant = getCustomEnchant(Arrays.copyOfRange(args, 1, args.length));
+        sender.sendMessage(TITLE + enchant.getDescription());
+      } catch (NullPointerException | IllegalArgumentException e) {
```

Comment:
It's bad practice to catch an exception that can easily be prevented with null checks.

Reply:
The goal is not to have an exceptions, not to have them occur discreetly